containerscan/datastructuresmethods.go

```diff
@@ -74,3 +74,32 @@ func (v *Vulnerability) IsRCE() bool {
 
 	return isRCE || strings.Contains(desc, "remote code execution") || strings.Contains(desc, "remote command execution") || strings.Contains(desc, "arbitrary code") || strings.Contains(desc, "code execution") || strings.Contains(desc, "code injection") || strings.Contains(desc, "command injection") || strings.Contains(desc, "inject arbitrary commands")
 }
+
+func (scanresult *ScanResultReportV1) Validate() bool {
```

Comment:
אולי כדאי לעטוף בinterface ולכתוב validator ו isRCE לאינטרפייס וזהו? (לא חובה-הצעה.)

Reply:
Bez input was not to put effort in V0 scan reports